metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures Microsoft Internet Information Services'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '6.8.1'
+version '6.8.2'
```

Comment:
We have tooling that auto bumps the versions and changelog so this isn't necessary